### PR TITLE
add support for $BROWSER environment variable

### PR DESCRIPTION
**Pull request by binarynoise**
Simplifies installation on systems with that environment variable set.

### CODE REVIEW

**Comment by woodruffw** [comment]
Thanks for the PR @binarynoise!

This seems reasonable to me. Perhaps it makes sense to spit out a little `stderr` notice as well, telling the user that we're using their `$BROWSER` setting?

File: install.sh [hunk 1]
```diff
@@ -3,6 +3,8 @@
 # Select browser
 if [[ -n $1 ]]; then
   browser=$1
+elif [[ -n $BROWSER ]]; then
+  browser=$(basename "$BROWSER")
```
Comment (woodruffw) [inline install.sh:7]: Extreme nitpick: unambiguous expansions:

```suggestion
elif [[ -n "${BROWSER}"]]; then
  browser=$(basename "${BROWSER}")
```

Comment (binarynoise) [reply install.sh]: I applied that. You missed the space before the `]]` though

**Comment by woodruffw** [comment]
Thanks @binarynoise!

**Pull request merged**